spec/support/acts_as_tenant.rb

```diff
@@ -9,10 +9,12 @@ def with_tenant(tenant, &proc)
     context "with tenant #{tenant.inspect}" do |*params|
       before(:each) do
         ActsAsTenant.current_tenant = send(tenant)
+        @old_host = Capybara.app_host
```

Comment:
suppose I have such code in my specs:
```
let!(:coursemology_instance) { create(:instance, host: 'coursemology.org' }
with_tenant(:instance) do
  let!(:course) { create(:course) }
  before { visit course_path(course) }
end

```

The above code will give me exception if there is no <code>Capybara.app_host = instance.host</code>

Because my course was created in <code>coursemology_instance</code>, and multi tenancy concern sets <code>current_tenant </code> to default_instance.